app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -187,7 +189,7 @@ def action_permission
 
       def load_and_authorize_plan
         @plan = load_dynflow_plan(params[:job_id])
-        return not_found(_('Report schedule not found, please ensure you have got right job_id')) if @plan.nil?
+        return not_found(_('Report not found, please ensure you have got right job_id')) if @plan.nil?
```

Comment:
`Report not found, please ensure you used the correct job_id` would I think sound better

Reply:
Done :)